### PR TITLE
Update ClickHouse to 19.14.7.15

### PR DESCRIPTION
**Pull request by gongled**
Hi, @andyone.

Please, update ClickHouse to the latest version.

### CODE REVIEW

File: specs/clickhouse/clickhouse.spec [hunk 1]
```diff
@@ -57,24 +57,21 @@
...
 Name:              clickhouse
-Version:           19.9.4.34
+Version:           19.14.7.15
 Release:           0%{?dist}
 License:           APL 2.0
 Group:             Applications/Databases
 URL:               https://clickhouse.yandex
 
 Source:            %{name}-%{version}.tar.bz2
-Source1:           %{name}.logrotate
```
Comment (andyone) [inline specs/clickhouse/clickhouse.spec:67]: Why did you remove logrotate config from the package?

File: specs/clickhouse/clickhouse.spec [hunk 2]
```diff
@@ -57,24 +57,21 @@
...
-
-Patch0:            %{name}-fix-tz-filepath.patch
```
Comment (andyone) [inline specs/clickhouse/clickhouse.spec:69]: Patch removed from spec but still exists in the repository.

**Review by andyone — CHANGES_REQUESTED** [review]

File: specs/clickhouse/clickhouse.spec (hunk 1 above)
Comment (andyone) [reply specs/clickhouse/clickhouse.spec:67]: @gongled ?

Comment (gongled) [reply specs/clickhouse/clickhouse.spec:67]: `logrotate` is corrupting logs due to ClickHouse rotates its logs itself.

Comment (andyone) [reply specs/clickhouse/clickhouse.spec:67]: Ok, thanks for the explanation.

**Review by andyone — APPROVED** [review]

File: specs/clickhouse/clickhouse.spec (hunk 2 above)
Comment (gongled) [reply specs/clickhouse/clickhouse.spec:69]: This patch is already merged into the codebase. I forgot to delete it from the `SOURCES` directory. Fixed (d9a58dc).

**Comment by andyone** [comment]
Looks good now, merged. Thanks!

**Comment by andyone** [comment]
The package was added to `kaos-testing` repository for CentOS 7.
